pyvista/core/filters/poly_data.py

```diff
@@ -1722,7 +1721,7 @@ def fill_holes(self, hole_size, inplace=False, progress_bar=False):  # pragma: n
         >>> assert edges.n_cells == 0  # doctest:+SKIP
 
         """
-        logging.warning('pyvista.PolyData.fill_holes is known to segfault. Use at your own risk')
+        warnings.warn('pyvista.PolyData.fill_holes is known to segfault. Use at your own risk')
```

Comment:
This type of warning will be annoying as it is thrown every time the filter is called.  Maybe move it into the docstring?

We could also have a special pyvista warning subclass for this kind of usage.  A user may have a use case where they know it is generally robust and would prefer to silence this warning.

Reply:
That's a great point! I like your idea; perhaps a `PyVistaPerformanceWarning`

Reply:
Done